CHANGELOG.md

```diff
@@ -1,11 +1,17 @@
 ## 5.5.1 (YYYY-MM-DD)
 
-### Bug Fixes
+### Enhancements
+* None
 
-* Building with Android App Bundle enabled should now work correctly (#5977).
+### Fixes
+* Building with Android App Bundle enabled should now work correctly [#5977](https://github.com/realm/realm-java/issues/5977, since v?.?.?).
 
-### Internal
+### Compatibility
+* File format: ver. 7 (upgrades automatically from previous formats)
+* Realm Object Server: 3.0.0 or later.
+* APIs are backwards compatible with all previous release of realm-java in the 5.x.y series.
```

Comment:
Isn't this redundant? I mean this is what semantic versioning means?

Reply:
We removed Breaking Changes as it's only used for the Major version releases (rare), so when we do a Major version it's included there. We added this comment under Compatibility to be explicit as we don't mention anywhere that we use Semantic Versioning :-)